sphinxgallery/tests/test_get_rst.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Author: Óscar Nájera
+# License: 3-clause BSD
+"""
+Testing the rst files generator
+"""
+from __future__ import division, absolute_import, print_function
+import sphinxgallery.gen_rst as sg
+from nose.tools import assert_equals
+
+
+def test_split_code_and_text_blocks():
+    """Test if a known example file gets properly split"""
+
```

Comment:
Something I forgot: testing that 19 hashes in a row does not produce a split would be good too.